jax/_src/pallas/fuser/block_spec.py

```diff
@@ -733,7 +733,7 @@ def new_index_map(i, *args):
   bcast_dim_block_shape = 1
   if isinstance(block_spec.block_shape[i], pallas_core.Element):
     bcast_dim_block_shape = pallas_core.Element(1)
-  new_block_shape = util.tuple_update(
+  new_block_shape = util.tuple_update(  # pytype: disable=wrong-arg-types
```

Comment:
Would it be sufficient to use `type: ignore[...]` comments instead of `pytype`? I am thinking in light of having a simpler convention for working around type checking issues.

Reply:
IMO it's `# type: ignore`

Reply:
JAX's codebase has ~550 instances of `type: ignore` and ~250 instances of `pytype: disable`.